types/node-forge/node-forge-tests.ts

```diff
@@ -145,6 +145,21 @@ if (forge.util.fillString('1', 5) !== '11111') throw Error('forge.util.fillStrin
     if (content.asd !== payload.asd) throw Error('forge.cipher.createCipher failed');
 }
 
+{
+    let count = 32;
+    let key = forge.random.getBytes(count, (err, bytes) => {
+        if (err) {
+            throw err;
+        }
+        if (bytes.length !== count) {
+            throw new Error('invalid length');
+        }
+        if (key !== bytes) {
+            throw new Error('return result and callback data not match');
```

Comment:
This test does not pass.

Reply:
I see…you’re adding this test in one commit and deleting it in the next? I guess we squash-merge here, so I won’t be picky about the commit discipline.